src/server_manager/web_app/digitalocean_server.ts

```diff
@@ -199,21 +229,23 @@ export class DigitalOceanServer extends ShadowboxServer implements server.Manage
     }
   }
 
-  // Gets the value for the given key, stored in the DigitalOcean tags.
-  private getTagValue(key: string): string {
-    const tagPrefix = makeKeyValueTagPrefix(key);
+  // Gets the key-value map stored in the DigitalOcean tags.
+  private getTagMap(): {[key: string]: string} {
```

Comment:
Please use a `Map` instead. It's safer and generally recommended over objects.

Reply:
Done